vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -93,20 +93,39 @@ type internal FSharpWorkspaceServiceFactory
                 match checkerSingleton with
                 | Some _ -> ()
                 | _ ->
-                    let checker = 
+                    let checker =
                         lazy
-                            let checker = 
+                            let editorOptions =
+                                let editorOptions = workspace.Services.GetService<EditorOptions>()
+
+                                match box editorOptions with
```

Comment:
May probably be just a [`Option.ofObj`](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-core-optionmodule.html#ofObj)

Reply:
It can't because then it would result in `Some (box editorOptions)` instead of `Some editorOptions`